### PR TITLE
WorkspaceOne SDK 24.11 release notes and document update

### PR DESCRIPTION
**Pull request by chaitu-venky**
https://github.com/euc-dev/ws1-sdk-uem-ios/issues/9.

Updated the required documentation notes for iOS SDK 24.11 release.

### CODE REVIEW

File: docs/integration/WorkspaceONE_iOS_BaseIntegration.pdf
Comment (helmlingp) [inline docs/integration/WorkspaceONE_iOS_BaseIntegration.pdf:1]: you have removed this file but it is referenced in https://github.com/euc-dev/ws1-sdk-uem-ios/blob/main/docs/index.md
please update https://github.com/euc-dev/ws1-sdk-uem-ios/blob/main/docs/index.md as well

File: docs/integration/WorkspaceONE_iOS_IntegrationPreparation.pdf
Comment (helmlingp) [inline docs/integration/WorkspaceONE_iOS_IntegrationPreparation.pdf:1]: you have removed this file but it is referenced in https://github.com/euc-dev/ws1-sdk-uem-ios/blob/main/docs/index.md please update https://github.com/euc-dev/ws1-sdk-uem-ios/blob/main/docs/index.md as required

File: docs/integration/WorkspaceONE_iOS_BaseIntegration.pdf
Comment (chaitu-venky) [reply docs/integration/WorkspaceONE_iOS_BaseIntegration.pdf:1]: The file is replaced: https://github.com/chaitu-venky/ws1-sdk-uem-ios/blob/main/docs/integration/WorkspaceONE_iOS_BaseIntegration.pdf, the same can be seen in forked repo.
Its updated.

File: docs/integration/WorkspaceONE_iOS_IntegrationPreparation.pdf
Comment (chaitu-venky) [reply docs/integration/WorkspaceONE_iOS_IntegrationPreparation.pdf:1]: The file is replaced: https://github.com/chaitu-venky/ws1-sdk-uem-ios/blob/main/docs/integration/WorkspaceONE_iOS_IntegrationPreparation.pdf, the same can be seen in forked repo.
Its updated.

**Review by helmlingp — APPROVED** [review]
I am unconvinced that the docs/integration/WorkspaceONE_iOS_BaseIntegration.pdf and docs/integration/WorkspaceONE_iOS_IntegrationPreparation.pdf files have been updated as the PR shows them removed and no new files.

**Pull request merged**